### ISSUE TITLE
Kstar bug on empty plans

### ISSUE THREAD

**Issue by TathagataChakraborti** [IBM/nl2flow#7]
Kstar errors out when there are no plans

```commandline
../../nl2flow/plan/planners.py:64: in plan
    result = planners.plan_unordered_topq(
../../../miniconda3/envs/lemming/lib/python3.10/site-packages/kstar_planner/planners.py:47: in plan_unordered_topq
    return run_planner(planner_args)
../../../miniconda3/envs/lemming/lib/python3.10/site-packages/kstar_planner/planners.py:25: in run_planner
    data = json.load(plans_file)
../../../miniconda3/envs/lemming/lib/python3.10/json/__init__.py:293: in load
    return loads(fp.read(),
../../../miniconda3/envs/lemming/lib/python3.10/json/__init__.py:346: in loads
    return _default_decoder.decode(s)
../../../miniconda3/envs/lemming/lib/python3.10/json/decoder.py:337: in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
```

🩹 I have temporarily put a try catch [here](https://github.com/IBM/nl2flow/blob/main/nl2flow/plan/planners.py#L74).

**Comment by ctpelok77** [comment]
@TathagataChakraborti please add the input files and your call

**Comment by ctpelok77** [comment]
Should be fixed in version 1.4.0. Please test @TathagataChakraborti
